cmd/init.go

```diff
@@ -999,6 +1001,8 @@ var initCmd = &cobra.Command{
 			flagCapellaPass, _ := cmd.Flags().GetString("capella-pass")
 			flagCapellaOid, _ := cmd.Flags().GetString("capella-oid")
 			flagCapellaOverrideToken, _ := cmd.Flags().GetString("capella-override-token")
+			flagCapellaInternalSupportToken, _ := cmd.Flags().GetString("capella-internal-support-token")
+			flagUploadServerLogsHostName, _ := cmd.Flags().GetString("upload-server-logs-host-name")
```

Comment:
It looks like you've added `flagUploadServerLogsHostName` here, but I don't see it being used below.